deploy/olm-catalog/ibm-block-csi-operator-community/1.3.0/ibm-block-csi-operator.v1.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,766 @@
+    mode: 420
+    filesystem: root
+    contents:
+    source: data:,%23%20Set%20SCSI%20command%20timeout%20to%20120s%20%28default%20%3D%3D
+    %2030%20or%2060%29%20for%20IBM%202145%20devices%0ASUBSYSTEM%3D%3D%22block%22%2C%20ACTION%3D%3D%22add
+    %22%2C%20ENV%7BID_VENDOR%7D%3D%3D%22IBM%22%2CENV%7BID_MODEL%7D%3D%3D%222145%22%2C%20RUN%2B%3D%22/bin/sh
+    %20-c%20%27echo%20120%20%3E/sys/block/%25k/device/timeout%27%22%0A
+    verification: {}
+    systemd:
+    units:
+    - name: multipathd.service
+    enabled: true
+    # Uncomment the following lines if this MachineConfig will be used with iSCSI connectivity
+    #- name: iscsid.service
+    #    enabled: true
+    ```
+
+    Apply the yaml file.
+    ```bash
+    oc apply -f 99-ibm-attach.yaml
+    ```
+
+    RHEL users should verify that the `systemctl status multipathd` output indicates that the multipath status is active and error-free.
+
+    ```bash
+    yum install device-mapper-multipath
+    modprobe dm-multipath
+    systemctl enable multipathd
+    systemctl start multipathd
+    systemctl status multipathd
+    multipath -ll
+    ```
+
+    ##### 2.2 Configuring for Kubernetes users (RHEL)
+    Create and set the relevant storage system parameters in the `/etc/multipath.conf` file. You can also use the default `multipath.conf` file, located in the `/usr/share/doc/device-mapper-multipath-*` directory.
+
+    Verify that the `systemctl status multipathd` output indicates that the multipath status is active and error-free.
+
+    ```bash
+    yum install device-mapper-multipath
+    modprobe dm-multipath
+    systemctl enable multipathd
+    systemctl start multipathd
+    systemctl status multipathd
+    multipath -ll
+    ```
+
+    #### 3. Configure storage system connectivity
+    3.1. Define the hostname of each Kubernetes node on the relevant storage systems with the valid WWPN(for Fibre Channel) or IQN(for iSCSI) of the node.
+
+    3.2. For Fibre Channel, configure the relevant zoning from the storage to the host.
+
+
+    ## Installation
+
+    ### Install the operator
+
+
+    #### 1. Download the manifest from GitHub.
+
+    ```bash
+    curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/master/deploy/installer/generated/ibm-block-csi-operator.yaml > ibm-block-csi-operator.yaml
+    ```
+
+    #### 2. (Optional): If required, update the image fields in the ibm-block-csi-operator.yaml.
+
+
+    #### 3. Create a namespace.
+
+    ```bash
+    $ kubectl create ns <namespace>
+    ```
+
+    #### 4. Install the operator, while using a user-defined namespace.
+
+    ```bash
+    $ kubectl -n <namespace> apply -f ibm-block-csi-operator.yaml
+    ```
+
+    ### Verify the operator is running:
+
+    ```bash
+    $ kubectl get pod -l app.kubernetes.io/name=ibm-block-csi-operator -n <namespace>
+    NAME                                    READY   STATUS    RESTARTS   AGE
+    ibm-block-csi-operator-5bb7996b86-xntss 2/2     Running   0          10m
```

Comment:
I think there is only one container in operator pod (not 2)

Reply:
done